migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator/sql_schema_calculator_flavour/postgres.rs

```diff
@@ -30,11 +28,6 @@ impl SqlSchemaCalculatorFlavour for PostgresFlavour {
         }
     }
 
-    fn default_native_type_for_scalar_type(&self, scalar_type: &ScalarType) -> serde_json::Value {
```

Comment:
Were these unused?

Reply:
yes, it's a datamodel connector method, so we were just calling flavour.connector().default_native_type_for_scalar_type()